cmd/av/stack_branch.go

```diff
@@ -193,9 +199,18 @@ func stackBranchMove(
 			Trunk: true,
 		}
 	}
+	currentMeta.PullRequest = nil
 	currentMeta.Name = newBranch
 	tx.SetBranch(currentMeta)
 
+	if !force {
+		if currentMeta.PullRequest != nil {
+			return errors.New(
+				"cannot rename branch because a pull request already exists (bypass with the `--force` flag, but this will require closing this pull request and opening a new one using `av pr create`)",
```

Comment:
This is a very long error message. It might make more sense to just print some output to stderr and return a shorter error message.
```go
_, _ = fmt.Fprint(os.Stderr,
    colors.Failure("Cannot rename branch <branchname>: pull request #<number> would be orphaned.\n"),
    colors.Faint("  - Use --force to override this check."),
)

return actions.ErrExitSilently{ExitCode: 127}
```